icommons_ext_tools/settings/base.py

```diff
@@ -79,7 +82,22 @@
 
 MANAGERS = ADMINS
 
-# DATABASES are defined in individual environment settings
+# DATABASES
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.oracle',
+        'NAME': SECURE_SETTINGS.get('db_default_name'),
```

Comment:
yeah, it's all a bit confusing but it's only the new way for a default postgres db.